gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -278,15 +274,15 @@ public int read(ByteBuffer byteBuffer) throws IOException {
     return bytesRead;
   }
 
-  private void requestObjectMedia(OptionalLong bytesToRead) throws IOException {
+  private void requestObjectMedia(@Nullable Integer bytesToRead) throws IOException {
```

Comment:
Do we definitely want to narrow this to an integer?

Reply:
Yes, because `ByteBuffer.remaining` that passed as a parameter here uses `int`, so there no sense to use `long`